recipes/qr-code-generator/all/test_package/CMakeLists.txt

```diff
@@ -4,7 +4,11 @@ project(test_package)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
+find_package(qr-code-generator REQUIRED)
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_link_libraries(${PROJECT_NAME} qr-code-generator::qr-code-generator)
+if (qr-code-generator_VERSION VERSION_LESS "1.7.0")
```

Comment:
I guess you need to pass this flag as an argument to cmake in the test package's conanfile.py.